easy-ui-react/src/HorizontalStack/HorizontalStack.tsx

```diff
@@ -83,10 +92,15 @@ export const HorizontalStack = React.forwardRef<null, HorizontalStackProps>(
         "wrap",
         wrap ? "wrap" : "nowrap",
       ),
+      ...getComponentToken(
+        "horizontal-stack",
+        "display",
+        inline ? "inline-flex" : "flex",
+      ),
     } as React.CSSProperties;
     return (
       <As
-        className={styles.HorizontalStack}
+        className={classNames(styles.HorizontalStack, className)}
         style={style}
         ref={ref}
         {...restProps}
```

Comment:
**question** it doesnt have to be part of this task,  but do you think at some point it might be worth adding some unit tests to the stack components?

Reply:
i think that's a good idea. i added a comment on the task for creating stories